src/code_translation/NodeSelector.py

```diff
@@ -28,12 +28,12 @@ class NodeSelector(cst.CSTVisitor):
     def __init__(self) -> None:
 
         self.variables: dict[str, Node] = {}
-        self.interesting_nodes: dict[cst.CSTNode, Node] = {}
+        self.interesting_nodes: dict[cst.Assign, Node] = {}
 
-        """Maps the library names in namespace to the responsible InputModule"""
         self.libraries: dict[str, InputModule] = {}
-        """Maps the method names in namespace to the responsible InputModule and the original method name."""
+        """Maps the library names in namespace to the responsible InputModule"""
         self.library_methods: dict[str, tuple[InputModule, str]] = {}
+        """Maps the method names in namespace to the responsible InputModule and the original method name."""
```

Comment:
yeah this just looks like garbage in the difference. I just move the docs below the variables where they are supposed to be

Reply:
no according to https://softwareengineering.stackexchange.com/questions/126601/comment-before-or-after-the-relevant-code

the docs should be above the code